Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift

```diff
@@ -99,10 +99,16 @@ public class ParleyComposeView: UIView {
     var placeholder: String? {
         didSet {
             placeholderLabel.text = placeholder
-            textView.accessibilityLabel = placeholder
+            textView.accessibilityLabel = placeholderVoiceOver ?? placeholder
```

Comment:
Shouldn't we just do a `replace("...", "")`?

Introducing a new string doesn't feel needed for this one

Reply:
Nevermind